src/grt/grt-wave_opt_file-parse-debug.adb

```diff
@@ -0,0 +1,64 @@
+--  GHDL Run Time (GRT) - Wave option file package for debugging.
+--  Copyright (C) 2016 Jonas Baggett
+--
+--  GHDL is free software; you can redistribute it and/or modify it under
+--  the terms of the GNU General Public License as published by the Free
+--  Software Foundation; either version 2, or (at your option) any later
+--  version.
+--
+--  GHDL is distributed in the hope that it will be useful, but WITHOUT ANY
+--  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+--  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+--  for more details.
+--
+--  You should have received a copy of the GNU General Public License
+--  along with GCC; see the file COPYING.  If not, write to the Free
+--  Software Foundation, 59 Temple Place - Suite 330, Boston, MA
+--  02111-1307, USA.
+--
+--  As a special exception, if other files instantiate generics from this
+--  unit, or you link this unit with other files to produce an executable,
+--  this unit does not by itself cause the resulting executable to be
+--  covered by the GNU General Public License. This exception does not
+--  however invalidate any other reasons why the executable file might be
+--  covered by the GNU Public License.
+
+-- Description: See package specifications
+
+with Grt.Astdio; use Grt.Astdio;
+
+package body Grt.Wave_Opt_File.Parse.Debug is
+
+   procedure Dump_Tree is
+   begin
+      for Index in Tree_Index_Type'Range loop
+         New_Line;
+         New_Line;
+         Put ("----------------------------");
+         New_Line;
```

Comment:
For Put + New_LIne, you can use Put_Line.
(Sorry I missed that in my first review).